mountpoint-s3/scripts/fs_latency_bench.sh

```diff
@@ -67,7 +72,14 @@ do
     iteration=10
     for i in $(seq 1 $iteration);
     do
-        /usr/bin/time -o ${results_dir}/time_output.txt -v ls -f "${target_dir}" >/dev/null 2>&1
+        # we don't know for sure how long does it take, but 5 minutes should be enough
```

Comment:
Was checking the previous benchmarks. max was around 10-11 seconds. So, 300 s should be fine.